src/main/java/lotto/model/Rank.java

```diff
@@ -0,0 +1,36 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+public enum Rank {
+    THIRD(3, 5000),
+    FOURTH(4, 50000),
+    FIFTH(5, 1500000),
+    SIXTH(6, 2000000000),
```

Comment:
```suggestion
    SIXTH(6, 2_000_000_000),
```

와 같이 가독성을 향상 시킬수 있습니다.